tests/timer.js

```diff
@@ -2,26 +2,39 @@ var Timer = require('../metrics/timer');
 
 var test = function(callback){
   console.log("\nCreating a new timer, updating 10 times / sec for 30 secs");
-  var timer = new Timer();
+  var timer = new Timer()
+      , n = 0;
+
+  var updateInterval = setInterval(function(){
+    //simulates the duration that a task takes between 0 and 500 milliseconds
+    //And then update the timer for the duration of that time
+    var duration = Math.round(Math.random() * (500));
```

Comment:
Added a duration here so that the timer makes more sense for others on how it is suppose to work and makes the output more "Ahhhhh, I see"